examples/authentication.rs

```diff
@@ -25,7 +25,7 @@ async fn main() -> Result<(), Error> {
         .await?;
 
     let positions = client
-        .call(GetPositionsRequest::futures(Currency::BTC))
+        .call(GetPositionsRequest::futures("BTC".to_string()))
```

Comment:
Changing from enum to String makes it require 1 more allocation right?? 
Is there any reason outside convenience for this change?